builtin/providers/aws/resource_aws_rds_cluster.go

```diff
@@ -71,6 +71,12 @@ func resourceAwsRDSCluster() *schema.Resource {
 				Computed: true,
 			},
 
+			"storage_encrypted": &schema.Schema{
```

Comment:
personally, I would suggest to be explicit in the terraform schema. This will safeguard against changes in AWS behaviour :)